qa/rpc-tests/sc_cert_listsinceblock.py

```diff
@@ -0,0 +1,381 @@
+        ch   = item['createdAtBlockHeight']
+
+        scid_swapped = str(swap_bytes(scid))
+
+        q = 10
+        blocks_d = {}
+        block_heights_d = {}
+        certs_d = {}
+        mat_height_d = {}
+
+        taddr1 = self.nodes[1].getnewaddress()
+        taddr2 = self.nodes[2].getnewaddress()
+
+        for i in range(3):
+
+            am_bwt1 = Decimal(i+1) + Decimal('0.01')
+            am_bwt2 = Decimal(i+1) + Decimal('0.02')
+            am_out  = Decimal('0.001') * (i+1)
+
+            mark_logs("Advance epoch...", self.nodes, DEBUG_MODE)
+            self.nodes[0].generate(EPOCH_LENGTH - 1)
+            self.sync_all()
+            epoch_number, epoch_cum_tree_hash = get_epoch_data(scid, self.nodes[0], EPOCH_LENGTH)
+ 
+            mark_logs("Node 1 sends a cert", self.nodes, DEBUG_MODE)
+            #==============================================================
+            pkh_arr = []
+            am_bwt_arr = []
+            raw_bwt_outs = {taddr1: am_bwt1, taddr2: am_bwt2}
+            for key in raw_bwt_outs.iterkeys():
+                pkh_arr.append(key)
+                am_bwt_arr.append(raw_bwt_outs[key])
+ 
+            proof = mcTest.create_test_proof(
+                "sc1", scid_swapped, epoch_number, q, MBTR_SC_FEE, FT_SC_FEE, epoch_cum_tree_hash, constant,
+                pkh_arr, am_bwt_arr)
+            
+            utx, change = get_spendable(self.nodes[1], CERT_FEE + am_out)
+            raw_inputs  = [ {'txid' : utx['txid'], 'vout' : utx['vout']}]
+            raw_outs    = { taddr1: change, taddr2: am_out}
+ 
+            raw_params = {
+                "scid": scid, "quality": q, "endEpochCumScTxCommTreeRoot": epoch_cum_tree_hash,
+                "scProof": proof, "withdrawalEpochNumber": epoch_number
+            }
+ 
+            try:
+                raw_cert    = self.nodes[1].createrawcertificate(raw_inputs, raw_outs, raw_bwt_outs, raw_params)
+                signed_cert = self.nodes[1].signrawtransaction(raw_cert)
+                certs_d[i] = self.nodes[1].sendrawtransaction(signed_cert['hex'])
+            except JSONRPCException as e:
+                errorString = e.error['message']
+                print("\n======> ", errorString)
+                assert_true(False)
+
+            self.sync_all()
+
+            mark_logs("cert = {}".format(certs_d[i]), self.nodes, DEBUG_MODE)
+            mat_height_d[i] = ch - 1 + (i+2)*elen + wlen
+            print "mat height = {}".format(mat_height_d[i])
+  
+            mark_logs("Node 0 generates 1 block", self.nodes, DEBUG_MODE)
+            bl = self.nodes[0].generate(1)[-1]
+            self.sync_all()
+            c = self.nodes[0].getblockcount()
+            blocks_d[i] = bl
+            block_heights_d[i] = c
+ 
+        # the first of the 3 certificates has reached maturity and Node2 has a consistent balance
+        bal = self.nodes[2].getbalance()
+        print "Node2 balance = {}".format(bal)
```

Comment:
Use the brackets for `print` for future compatibility with Python 3.

```suggestion
        print("Node2 balance = {}".format(bal))
```